central/reports/scheduler/schedule.go

```diff
@@ -72,12 +73,12 @@ var (
     }`
 
 	vulnReportEmailTemplate = `
```

Comment:
I just realized that this and other strings here are put in `var` block. They can and should be `const`-s. That's not an issue that you introduced, but could you please fix it?
`log` and `scheduledCtx` would have to stay `var`-s because Go support for constants/read-onlys is limited to only primitive types.

Reply:
Okay, sure.